app/controllers/admin/trips_controller.rb

```diff
@@ -1,9 +1,54 @@
 module Admin
   class TripsController < BaseController
+    before_action :set_trip, only: [:edit, :update, :destroy]
```

Comment:
Good job on doing this. I'll try and make the stations have a before action also for consistency